### ISSUE TITLE
Custom timestamps column names

### ISSUE THREAD

**Issue by mojidabckuu** [zdennis/activerecord-import#400]
I have custom column names for my timestamps.
When I try `Class.import` import functions tries to fill `created_at`, `created_on` attributes with lambda. The responsible properties names are
```
AREXT_RAILS_COLUMNS = {
      create: { "created_on" => tproc,
                "created_at" => tproc },
      update: { "updated_on" => tproc,
                "updated_at" => tproc }
    }.freeze
```
But it doesn't take in scope `timestamp_attributes_for_create` from ActiveRecord::Base.
Is there any chance to add more ext_rails_columns ?

**Comment by jkowens** [comment]
I think that seems like a good suggestion, to make use of the `timestamp_attributes_for_create` and `timestamp_attributes_for_update` methods on ActiveRecord::Base.

How are you defining the custom timestamp columns?  Are you monkey patching the ActiveRecord::Timestamp module methods?

**Comment by mojidabckuu** [comment]
@jkowens 
Yeah, I know that monkey patching is an evil and we are the warriors against the darkness, but for me it was the best suitable solution to keep my migrations the same regardless `timestamps` function body. Anyway I could replace all the calls of `timestamps` with en extended custom method for `ActiveRecord::ConnectionAdapters::TableDefinition` something like `timestamps_ex`

**Comment by jkowens** [comment]
@mojidabckuu I just merged #401 into master which I think should resolve your issue. Are you able to give it a spin and let me know?

**Comment by mojidabckuu** [comment]
@jkowens I've checked and it worked for me 👌

**Comment by jkowens** [comment]
This is included in the v0.18.0 release.
